gammapy/catalog/tests/test_hess.py

```diff
@@ -76,3 +78,32 @@ def test_ecut_error(self):
 
         energy_err = err / val ** 2
         assert_allclose(err, energy.std_dev)
+
+    @requires_dependency('matplotlib')
+    def test_model_plot(self):
+        model = self.source.spectral_model
+        erange = Quantity([1, 10], 'TeV')
+        model.plot(erange)
+
+    def test_model(self):
+        model = self.source.spectral_model
+        pars = model.parameters
+        assert_quantity_allclose(pars.amplitude, Quantity(1.716531924e-11, 'TeV-1 cm-2 s-1'))
+        assert_quantity_allclose(pars.index, Quantity(2.3770857316, ''))
+        assert_quantity_allclose(pars.reference, Quantity(1.1561109149, 'TeV'))
+
+        emin, emax = Quantity([1, 1E10], 'TeV')
+        desired =  Quantity(self.source.data['Flux_Spec_PL_Int_1TeV'], 'cm-2 s-1')
+        assert_quantity_allclose(model.integral(emin, emax), desired, rtol=0.01)
+
+    def test_ecpl_model(self):
+        model = self.cat['HESS J0835-455'].spectral_model
+        pars = model.parameters
+        assert_quantity_allclose(pars.amplitude, Quantity(6.408420542586617e-12, 'TeV-1 cm-2 s-1'))
+        assert_quantity_allclose(pars.index, Quantity(1.3543991614920847, ''))
+        assert_quantity_allclose(pars.reference, Quantity(1.696938754239, 'TeV'))
+        assert_quantity_allclose(pars.lambda_, Quantity(0.081517637, 'TeV-1'))
+
+        emin, emax = Quantity([1, 1E10], 'TeV')
+        desired =  Quantity(self.source.data['Flux_Spec_PL_Int_1TeV'], 'cm-2 s-1')
```

Comment:
Here you're comparing an ECPL model flux to `Flux_Spec_PL_Int_1TeV`.
Shouldn't this be comparing to the ECPL flux from the catalog?
Does the test pass locally for you? Maybe the `rtol` has to be decreased a bit to see that difference?